pom.xml

```
@@ -97,6 +97,21 @@
             <artifactId>ion-java-path-extraction</artifactId>
             <version>1.2.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.amazon.ion</groupId>
+            <artifactId>ion-element</artifactId>
+            <version>1.3.0</version>
```

As with `ion-java`, let's take the latest available version.
```suggestion
            <version>[1.3.0, )</version>
```